ntbs-service/Services/SpecimenService.cs

```diff
@@ -156,19 +168,19 @@ public async Task UnmatchSpecimen(int notificationId, string labReferenceNumber,
             await _auditService.AuditUnmatchSpecimen(notificationId, labReferenceNumber, userName);
         }
 
-        public async Task MatchSpecimenAsync(int notificationId, string referenceLaboratoryNumber, string userName)
+        public async Task MatchSpecimenAsync(int notificationId, string labReferenceNumber, string userName)
         {
             using (var connection = new SqlConnection(_specimenMatchingDbConnectionString))
             {
                 connection.Open();
                 await connection.QueryAsync(
                     "uspMatchSpecimen",

```

Comment:
uspMatchSpecimen returns a 0, or 1 based on success of match. 
We could potentially lean on that here as the import process assumes a good deal of external logic works correctly. If we're changing match we could also look at changing unmatch too.